File: python/program/Food.py

class Food:
    def __init__(self, brand="", type=""):
        self.__brand = brand
        self.__type = type
    
    def set_brand(self, brand):
        self.__brand = brand
    
    def set_type(self, type):
        self.__type = type
    
    def get_brand(self):
        return self.__brand
    
    def get_type(self):
        return self.__type

File: cpp/program/Food.cpp

#pragma once
#include <iostream>
#include <string>

using namespace std;

class Food {
private:
    string brand;
    string type;
public:
    Food() {
    }

    Food(string brand, string type) {
        this->brand = brand;
        this->type = type;
    }

    void setBrand(string brand) {
        this->brand = brand;
    }

    void setType(string type) {
        this->type = type;
    }

    string getBrand() {
        return this->brand;
    }

    string getType() {
        return this->type;
    }

    ~Food() {
    }
};